Dockerfile

```diff
@@ -1,11 +1,37 @@
-FROM ubuntu-debootstrap:14.04
+FROM quay.io/deis/go-dev:0.9.0
 
 ENV DOCKERIMAGE=1
 
 RUN apt-get update -y && apt-get install -y curl openssh-client git
-RUN curl -sSL http://deis.io/deis-cli/install-v2-alpha.sh | bash && mv ./deis /bin/deis
-COPY tests/tests.test .
-RUN mv tests.test /bin
-RUN mkdir /files
-COPY tests/files /files
-CMD /bin/tests.test
+# RUN curl -sSL http://deis.io/deis-cli/install-v2-alpha.sh | bash && mv ./deis /bin/deis
+# COPY tests/
+# RUN mv tests.test /bin
+# RUN mkdir /files
+# COPY tests/files /files
+# CMD /bin/tests.test
+
+ENV GO15VENDOREXPERIMENT=1
```

Comment:
If our base image is `go-dev` above, we shouldn't need lines 13-26 or lines 30-34 (https://github.com/deis/docker-go-dev/blob/master/rootfs/Dockerfile as reference)